exporter/datadogexporter/factory.go

```diff
@@ -39,6 +40,13 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry"
 )
 
+var logsAgentExporterFeatureGate = featuregate.GlobalRegistry().MustRegister(
+	"exporter.datadogexporter.logsagentexporter",
```

Comment:
We got it wrong for the metric export native client, but the usual way of naming feature gates is to (1) use UpperCamelCase for the last part and (2) make it a sentence.

```suggestion
	"exporter.datadogexporter.UseLogsAgentExporter",
```